src/pool/rpc.h

```diff
@@ -366,6 +369,18 @@ CRT_RPC_DECLARE(pool_get_acl, DAOS_ISEQ_POOL_GET_ACL, DAOS_OSEQ_POOL_GET_ACL)
 
 CRT_RPC_DECLARE(pool_prop_set, DAOS_ISEQ_POOL_PROP_SET, DAOS_OSEQ_POOL_PROP_SET)
 
+#define DAOS_ISEQ_POOL_LIST_CONT	/* input fields */		 \
+	((struct pool_op_in)	(plci_op)			CRT_VAR) \
+	((crt_bulk_t)		(plci_cont_bulk)		CRT_VAR) \
+	((uint64_t)		(plci_ncont)			CRT_VAR)
+
+#define DAOS_OSEQ_POOL_LIST_CONT	/* output fields */		 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
Will keep the macro as it is coded, to be consistent with the other examples in the file.